### PR TITLE
use options.as to determine Element in itemView

### PR DESCRIPTION
**Pull request by AleksMWix**

### CODE REVIEW

**Review by yoavtsook2806 — COMMENTED** [review]
Hey, why do you need this option? 
What are you sending there instead of div?

**Review by yoavtsook2806 — COMMENTED** [review]
1. Please change the prop to an object named elementsOveride with one property at the moment: {item: ${yourElement}}. this prop should not be under 'options', but as a separated prop.
2. Please add this API in another PR to master as well.
3. This is the last PR which is not a bug fix that we approve for version 4 :)

**Comment by AleksMWix** [comment]
#rebuild

File: packages/gallery/src/components/item/itemView.js [hunk 1]
```diff
@@ -987,8 +987,9 @@ class ItemView extends React.Component {
       options.titlePlacement !== GALLERY_CONSTS.placements.SHOW_ON_HOVER &&
       !this.hasRequiredMediaUrl;
     const itemAriaRole = this.getItemAriaRole();
+    const Element = this.props.elementsOverride.item || 'div';
```
Comment (yoavtsook2806) [inline packages/gallery/src/components/item/itemView.js]: should add '?' for this check. if elementOverride doesn't exist this line will access undefined.
something like
    const Element = this.props?.elementsOverride.item ?? 'div';

**Review by yoavtsook2806 — APPROVED** [review]

**Pull request merged**